extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -122,8 +126,8 @@ const defaultInstallationCommands = [
 	},
 	{
 		comment: localize('resourceDeployment.Azdata.InstallingAzdata', "installing azdata ..."),
-		command: `pip3 install -r https://aka.ms/azdata --quiet --user`
+		command: `pip3 install -r ${vscode.workspace.getConfiguration(DeploymentConfigurationKey)[AzdataPipInstallUriKey]} ${vscode.workspace.getConfiguration(DeploymentConfigurationKey)[azdataPipInstallArgsKey]} --quiet --user`
```

Comment:
Now that this is user configurable you should probably be quoting the URI. (the args should be done in the config so shouldn't be needed)

Reply:
the main use case for this is for internal testing, and the url we use do not have spaces, I am going to leave this as is to avoid unexpected behavior on different platforms.